docs/decisions/0010-limit-modulestore-use-in-lms.rst

```diff
@@ -0,0 +1,273 @@
+Limit LMS Modulestore access to the courseware app
```

Comment:
The courseware app is called out as the big exception here, but it isn't mentioned anywhere why.

I think I know why (rendering the unit, yeah?) but could you give it a sentence or two in the ADR? And what do you see as the big blocker towards resolving it?

Reply:
Ah right, makes sense. Will add.